src/main/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/repository/MatchRepository.java

```diff
@@ -2,12 +2,15 @@
 
 import ch.uzh.ifi.hase.soprafs22.screwyourneighborserver.entity.Match;
 import ch.uzh.ifi.hase.soprafs22.screwyourneighborserver.entity.MatchEmbedProjection;
-import java.util.List;
 import org.springframework.data.jpa.repository.JpaRepository;
-import org.springframework.data.repository.query.Param;
 import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+import org.springframework.data.rest.core.annotation.RestResource;
+import org.springframework.lang.NonNull;
 
 @RepositoryRestResource(excerptProjection = MatchEmbedProjection.class)
 public interface MatchRepository extends JpaRepository<Match, Long> {
-  List<Match> findAllByMatchNumber(@Param("matchNumber") int matchNumber);
+  @Override
+  @NonNull
+  @RestResource(exported = false)
+  <S extends Match> S save(@NonNull S entity);
```

Comment:
Wenn wir das machen, was gnau ist nun noch exposed als Endpoint?

Reply:
Get collection and get item